.github/ISSUE_TEMPLATE.md

```diff
@@ -18,4 +18,4 @@ x.y.z
 
 ### Testcase
 [Fork, update, and replace this pen to show the bug]:
-http://codepen.io/levithomason/pen/ZpBaJX
+https://codesandbox.io/s/2l3n74j9y
```

Comment:
`codesandbox` is more modern tool, there are real NPM deps and `prettier` :heart: @levithomason Feel free fo fork it, update and replace link if you will want to update something